lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -0,0 +1,134 @@
+require 'active_support/core_ext/module/delegation'
+require 'pathname'
+
+module ManageIQ
+  module ApplianceConsole
+    class MessageConfiguration
+      attr_reader :username, :password,
+                  :miq_config_dir_path, :config_dir_path, :sample_config_dir_path,
+                  :tools_log4_properties_path, :tools_log4_properties_sample_path,
+                  :client_properties_path,
+                  :keystore_dir_path, :truststore_path, :keystore_path,
+                  :messaging_yaml_sample_path, :messaging_yaml_path
+
+      BASE_DIR                          = "/opt/kafka".freeze
+      LOGS_DIR                          = "#{BASE_DIR}/logs".freeze
+      CONFIG_DIR                        = "#{BASE_DIR}/config".freeze
+      SAMPLE_CONFIG_DIR                 = "#{BASE_DIR}/config-sample".freeze
+      MIQ_CONFIG_DIR                    = "/var/www/miq/vmdb/config".freeze
```

Comment:
Should this use `ManageIQ::ApplianceConsole::RAILS_ROOT.join("config")`?

Reply:
Good point. I didn't even know that existed.
Addressed by commit: https://github.com/ManageIQ/manageiq-appliance_console/pull/130/commits/9bb79759c88dddb951dad05049193da90aaacbcc